### ISSUE TITLE
[Landroid V4] Wrong value for current-blade-time channel

### ISSUE THREAD

**Issue by sihui62** [nibi79/worxlandroid#89]
OH version: openHAB 4.0.0 Build #3544 
Binding version: 4.0.0.202307021742

Issue:
Channel `current-blade-time` and channel `total-blade-time` show the same value.
Channel `total-blade-time` seems to be correct (compared to the Android app), channel `current-blade-time` is incorrect.

Config:
```
Number:Time                Worx_TotalBladeTime       "Total blade time [%.0f h]"               { unit="h", channel="worxlandroid:mower:mower:m500plus:metrics#total-blade-time" }
Number:Time                Worx_CurrentBladeTime     "Current blade time [%.0f h]"             { unit="h", channel="worxlandroid:mower:mower:m500plus:metrics#current-blade-time" }
```
Worx_CurrentBladeTime shows as 730 h,
Worx_TotalBladeTime shows as 730 h.

**Comment by clinique** [comment]
You're correct, this is not fully functional (lines 573-578). I will investigate how it was handle previously - what's for sure is that currently `blade_work_time_reset` is never set to any value.

**Comment by sihui62** [comment]
> what's for sure is that currently blade_work_time_reset is never set to any value.

Yes, same issue in the old version. I had to use `Item.postUpdate(0)` to reset the blade time  :-)

**Comment by sihui62** [comment]
OH version: openHAB 4.0.2 Release
Binding version: 4.1.0.202309261327

Unfortunately I am seeing the same issue from the first post above. I cleared cache and tmp folders after installation of the newest binding version.

Channel `current-blade-time` and channel `total-blade-time` show the same value:

openHAB:
![oh1](https://github.com/nibi79/worxlandroid/assets/10405486/5e7f4736-604c-4ac0-b604-7a4325d3451b)
![oh2](https://github.com/nibi79/worxlandroid/assets/10405486/f5c51559-29c0-4c79-b335-56f7772695c6)


Android App:
![app](https://github.com/nibi79/worxlandroid/assets/10405486/d0b40ae0-bae2-4425-a459-570ae3cd8a7e)

**Comment by clinique** [comment]
Have you tried "Klingenarbeitszeit.sendCommand(0)" ?

**Comment by sihui62** [comment]
> Have you tried

No, but I did now:

openHAB: Item state is 0 h
Android App (after refreshing): 115 h

Then I polled the AWS:
openHAB: Item state is 0 h
Android App (after refreshing): 115 h

So it seems they are out of sync.

**Comment by clinique** [comment]
I guess that the logic is stored in the android app, not provided by the API. Was it functioning in version 3 ?

**Comment by clinique** [comment]
> > Was it functioning in version 3 ?
> 
> Yes.

Not sure as you would expect being synced with the Android App. I had a look at the code of v3 and I'm now doing the same in v4. No data seem to come from the API for these 2 values.

**Comment by clinique** [comment]
I found how to make it work in ioBroker ! Will add it.

**Comment by sihui62** [comment]
> Will add it.

With binding version 4.1.0.202310301404 all mentioned channels are correctly populated, thanks a lot.
From my point of view this issue can be closed.

**Comment by clinique** [comment]
@sihui62 : I can not close it. I leave it to you.

**Comment by sihui62** [comment]
Fixed with 4.1.0.202310301404
